core/src/main/java/org/apache/accumulo/core/conf/Property.java

```diff
@@ -1165,7 +1167,11 @@ public enum Property {
   @Experimental
   COMPACTION_COORDINATOR_TSERVER_COMPACTION_CHECK_INTERVAL(
       "compaction.coordinator.tserver.check.interval", "1m", PropertyType.TIMEDURATION,
-      "The interval at which to check the tservers for external compactions.", "2.1.0");
+      "The interval at which to check the tservers for external compactions.", "2.1.0"),
```

Comment:
Noticed some of the existing props were unused when looking at this PR and opened #4932